### DISCUSSION TITLE
Secret Signature Token is always the same?

### DISCUSSION THREAD

**Discussion by ms4sman** [YOURLS/YOURLS#3758] [General]
Hi all. I feel like I'm missing something here, but every time I make an instance of Yourls, the Secret Signature Key is the same as the last instance I made. Is this how it's meant to be?

I'm using the one-click app template for Yourls for Caprover (https://wizardly-ptolemy-8fcac8.netlify.app/app/yourls) which sets up a Docker instance of Yourls. I've made sure with each of my installs to use a new and unique cookie key value. Since according to here (https://yourls.org/docs/guide/advanced/passwordless-api#reset-your-secret-signature-token), that cookie key is how the signature key is created. 

What am I doing wrong here that is causing the signature token to always be the same? Thanks!

**Comment by dgw** [comment]
I don't see where [the defined cookiekey variable](https://github.com/caprover/one-click-apps/blob/7ff4ee9ebc08e1fd54b566e069d2288b7eacb24b/public/v4/apps/yourls.yml#L45-L47) is actually used anywhere else in the CapRover config.

You should bring this up to whomever maintains the one-click configuration. The cookie key isn't the only variable that appears to be defined but never used elsewhere.

**Comment by ms4sman** [comment]
Ah, thanks for the response! I wondered if perhaps it was something like that. I'm not familiar enough with their one-click apps to have dug in and found that, so thanks!

I'll try bringing it up with them, now that I know its with their config!

Thank you!
